website/content/en/docs/building-operators/ansible/reference/scaffolding.md

```diff
@@ -21,28 +21,20 @@ table describes a basic rundown of each generated file/directory.
 | File/Folders   | Purpose |
 | :---           | :---    |
 | Dockerfile | The Dockerfile for building the container image for the operator. |
-| Makefile | Contains make targets for building, publishing, deploying the
-container image that wraps the operator binary, and make targets for
-installing and uninstalling the CRD. |
+| Makefile | Contains make targets for building, publishing, deploying the container image that wraps the operator binary, and make targets for installing and uninstalling the CRD. |
 | PROJECT | A YAML file containing meta information for the operator. |
 | config/crd | The base CRD files and the kustomization settings. |
 | config/default | Collects all operator manifests for deployment, used by `make deploy`. |
 | config/manager | The controller manager deployment. |
-| config/prometheus | The ServiceMonitor resource for monitoring the
-operator. |
-| config/rbac | The role, role binding for leader election and authentication
-proxy. |
+| config/prometheus | The ServiceMonitor resource for monitoring the operator. |
+| config/rbac | The role, role binding for leader election and authentication proxy. |
```

Comment:
```suggestion
| config/rbac | Contains the manifests used to configure the RBCA permissions |
```

Reply:
I dont want to scope creep into changing this content-- but since I linked to it I wanted to at least make sure the table renders correctly.